packages/api-clients/open-api/tenantApi.yml

```diff
@@ -996,6 +996,26 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+    delete:
```

Comment:
This entire path seems to be missing the correlationId, which in our process specs should be present in all routes 👇  Not only the delete, but the entire `/tenants/delegatedProducer:` path, see other paths for reference

```
parameters:
  - $ref: "#/components/parameters/CorrelationIdHeader"
```

Reply:
Great catch!